app/views/reports/setup.js.erb

```diff
@@ -21,7 +21,6 @@ $('#defined_reports_step_2').html("<%= raw escape_javascript(render(:partial =>
 $('#report_selection').hide();
 $('#defined_reports, #defined_reports_step_2').show();
 // $('#defined_reports_step_1').dialog('close');
-$("select").selectpicker()
```

Comment:
Sam had added this to initialize selectpickers. Maybe in the future we should add the selectpicker class to the reports selectpickers and add $('.selectpicker').selectpicker() here